### ISSUE TITLE
Change knob/slider/slider label color when muted.

### ISSUE THREAD

**Issue by pkim168** [nvdweem/PCPanel#22]
Not sure if the ability to do this already exists. If an knob/slider/slider label controls an application, have to option to change to a different color when it is muted.

**Comment by nvdweem** [comment]
It doesn't work perfectly, but it is already possible via the lighting config > Mute override option:
![image](https://user-images.githubusercontent.com/830783/206098962-da88778e-56d8-464e-98a8-5a833ba05fcc.png)

I've seen the software keep the override color when it's closed while muted and then stay on the override color whether it's muted or not, that's something I want to fix soon.

**Comment by pkim168** [comment]
I've tried using this but I still don't understand how it's supposed to work. I've only managed to have the rgb change once but I had to select a specific device for it instead of using default (I switch between headphones and speakers) and when I did that it would only change if the device itself was muted, not the app.

**Comment by nvdweem** [comment]
I agree that the label isn't super clear, but what it tries to say is that the default will check the mute color for whatever the dial/slider (specifically not the button) is controlling.
So, with a dial that controls the default device volume, the default 'Follow what is controlled' option should switch the color when the default device is muted.

If your dial does not control volume of the default device there are the specific devices in the dropdown, but not the default device (that could be an improvement as well).

For me I have 1 dial for which the click action is 'Mute default device' and the turn action is 'Default device volume'. It's green with a mute override color of red. Clicking the button or muting the device in a different way will then make the button red.

I'm not sure if you are using Windows though because I'm not sure if the feature works on the Linux version.

**Comment by pkim168** [comment]
Muting devices works, but changing the rgb when muting apps doesnt.

**Comment by pkim168** [comment]
Is there a way to change the rgb based on if an OBS source is muted?

**Comment by nvdweem** [comment]
No, that's not possible yet. I hadn't expected you to mean an OBS source when you asked about an application being muted, I thought you meant an entire application.

**Comment by nvdweem** [comment]
I attempted to add this feature a few days ago. The OBS sources aren't currently listed in the override dropdown but it should work if the dial/slider controls the source volume.
